### PR TITLE
Use *.toml files provided in aktualizr github repo

### PR DESCRIPTION
**Pull request by OYTIS**
This enables backwards incompatible changes to configuration format.

Tested for autoprovisioning case on qemu.

### CODE REVIEW

File: recipes-sota/aktualizr/aktualizr_git.bb [hunk 1]
```diff
@@ -40,6 +40,10 @@ do_install_append_class-target () {
 do_install_append_class-native () {
     rm -f ${D}${bindir}/aktualizr
     rm -f ${D}${bindir}/aktualizr-info
+    install -d ${D}${libdir}/sota
+    install -m 0644 ${S}/config/sota_autoprov.toml ${D}/${libdir}/sota/sota_autoprov.toml
+    install -m 0644 ${S}/config/sota_hsm_test.toml ${D}/${libdir}/sota/sota_hsm_test.toml
+    install -m 0644 ${S}/config/sota_implicit_prov.toml ${D}/${libdir}/sota/sota_implicit_prov.toml
```
Comment (patrickvacek) [inline recipes-sota/aktualizr/aktualizr_git.bb]: This means all of the important configs will be installed on every device, right? I think I actually like that.

File: recipes-sota/aktualizr/aktualizr-auto-prov.bb [hunk 2]
```diff
@@ -40,7 +39,7 @@ do_install_append() {
         install -d ${D}/${systemd_unitdir}/system
         install -m 0644 ${WORKDIR}/aktualizr-autoprovision.service ${D}/${systemd_unitdir}/system/aktualizr.service
         install -d ${D}${libdir}/sota
-        install -m "0644" ${WORKDIR}/sota_autoprov.toml ${D}${libdir}/sota/sota.toml
+        install -m "0644" ${STAGING_DIR_NATIVE}/${libdir}/sota/sota_autoprov.toml ${D}${libdir}/sota/sota.toml
```
Comment (cajun-rat) [inline recipes-sota/aktualizr/aktualizr-auto-prov.bb]: What about creating a symlink to the thing installed by the aktualizr recipe? Adding a dependency on aktualizr-native feels a bit weird

File: recipes-sota/aktualizr/aktualizr_git.bb (hunk 1 above)
Comment (OYTIS) [reply recipes-sota/aktualizr/aktualizr_git.bb]: Um... No, that's actually only for native installation. They are later used by provisioning recipes. But we might have this option as well.

File: recipes-sota/aktualizr/aktualizr-auto-prov.bb (hunk 2 above)
Comment (OYTIS) [reply recipes-sota/aktualizr/aktualizr-auto-prov.bb]: You mean install all of the *.toml files on every device and then symlink to the correct one?

File: recipes-sota/aktualizr/aktualizr_git.bb (hunk 1 above)
Comment (patrickvacek) [reply recipes-sota/aktualizr/aktualizr_git.bb]: Ah, now I see. I read too fast and didn't see the context that that was only in the `-native` part.

**Comment by patrickvacek** [comment]
Looks like this needs a rebase before merging. But otherwise seems good to me.

**Comment by OYTIS** [comment]
Thank you, rebased.

**Pull request merged**